add-custom-dynamic-entity-data-source/src/main/java/com/esri/arcgismaps/sample/addcustomdynamicentitydatasource/screens/MainScreen.kt

```diff
@@ -89,16 +116,6 @@ fun MainScreen(sampleName: String) {
                         text = stringResource(if (!isConnected) R.string.connect else R.string.disconnect)
                     )
                 }
-                // Display a MessageDialog with identify information.
-                mapViewModel.messageDialogVM.apply {
-                    if (dialogStatus) {
-                        MessageDialog(
-                            title = messageTitle,
-                            description = messageDescription,
-                            onDismissRequest = ::dismissDialog
-                        )
-                    }
-                }
```

Comment:
I believe you would need to add back the `MessageDialog` Composable on your MainScreen here.